src/bom/bitcode.rs

```diff
@@ -640,6 +640,20 @@ fn build_bitcode_tar(bc_target : &OsString,
     Ok(())
 }
 
+
+fn bitcode_hashval(cwd : &Path, bc_target : &OsString) -> anyhow::Result<String> {
```

Comment:
I think it would be more conventional for this to be `bc_target: &OsStr` (which is like `str` to `String`)